vscode/oberon0lsp/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "oberon0lsp",
+  "displayName": "Oberon0 LSP",
+  "description": "Plugin for Oberon0 Language Server Protocol support",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.67.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/melt-umn/Oberon0.git"
+  },
+  "publisher": "MELT",
+  "categories": [
+    "Programming Languages"
+  ],
+  "tags": [
+    "extensible language"
+  ],
+  "license": "LGPL-3.0-or-later",
+  "activationEvents": [
+    "onLanguage:Oberon0"
+  ],
+  "main": "./out/extension.js",
+  "contributes": {
+    "languages": [
+      {
+        "id": "Oberon0",
+        "aliases": [
+          "Oberon0",
+          "ob"
+        ],
+        "extensions": [
+          ".ob"
+        ],
+        "configuration": "./language-configuration.json"
+      }
+    ],
+    "configuration": [
+      {
+        "title": "Oberon0",
+        "properties": {
+          "Oberon0.compilerJar": {
+            "type": "string",
+            "default": "oberon0_a5.jar",
+            "description": "Path to the jar containing an alternate version of the silver-oberon0 compiler. This can be a relative path from the first workspace folder."
```

Comment:
> 
```suggestion
            "description": "Path to the jar containing an alternate version of the oberon0 driver. This can be a relative path from the first workspace folder."
```